### PR TITLE
Implement cv.imread & cv.imshow and insert interactive pages in tutorials

### PR DESCRIPTION
**Pull request by pancx**
1. insert the old interactive pages in markdown by using \htmlonly and \endhtmlonly command.
2. delete the useless interactive page
3. rename js_interactive_tutorials to js_assets to put some images needed in
4. add helper.js which contains functions of imread and imshow, insert it to output binary by using --post-js command
5. change imread & imshow to cv.imread & cv.imshow in tutorials

### CODE REVIEW

**Comment by pancx** [comment]
@huningxin  please review

File: doc/CMakeLists.txt [hunk 1]
```diff
@@ -181,30 +181,30 @@ if(BUILD_DOCS AND DOXYGEN_FOUND)
...
   set(ocv_js_dir "${CMAKE_BINARY_DIR}/bin")
   set(ocv_js "opencv.js")
 
-  set(opencv_interactive_tutorials_deps "")
+  set(js_tutorials_assets_deps "")
 
   # make sure the build directory exists
   file(MAKE_DIRECTORY "${opencv_tutorial_html_dir}")
 
   # gather and copy specific files for js test
```
Comment (huningxin) [inline doc/CMakeLists.txt]: The commend is not accurate.

File: doc/CMakeLists.txt [hunk 2]
```diff
@@ -166,7 +166,7 @@ if(BUILD_DOCS AND DOXYGEN_FOUND)
 
   add_custom_target(doxygen
     COMMAND ${DOXYGEN_EXECUTABLE} ${doxyfile}
-    DEPENDS copy_js_interactive_tutorials ${doxyfile} ${rootfile} ${bibfile} ${deps}
+    DEPENDS copy_js_assets ${doxyfile} ${rootfile} ${bibfile} ${deps}
```
Comment (huningxin) [inline doc/CMakeLists.txt]: I think it should depends on assets file and opencv.js file.

File: doc/CMakeLists.txt [hunk 3]
```diff
@@ -181,30 +181,30 @@ if(BUILD_DOCS AND DOXYGEN_FOUND)
...
 
   # add dep on opencv.js target
-  list(APPEND opencv_interactive_tutorials_deps ${ocv_js})
+  list(APPEND js_tutorials_assets_deps "${ocv_js_dir}/${ocv_js}")
 
-  add_custom_target(copy_js_interactive_tutorials
+  add_custom_target(copy_js_assets
                     COMMAND ${CMAKE_COMMAND} -E copy_if_different "${ocv_js_dir}/${ocv_js}" "${opencv_tutorial_html_dir}/${ocv_js}"
-                    DEPENDS ${opencv_interactive_tutorials_deps})
+                    DEPENDS ${js_tutorials_assets_deps})
```
Comment (huningxin) [inline doc/CMakeLists.txt]: why depends on assets? it should only depend on `${ocv_js_dir}/${ocv_js}`, correct?

File: modules/js/CMakeLists.txt [hunk 4]
```diff
@@ -44,6 +44,8 @@ set(bindings_cpp "${CMAKE_CURRENT_BINARY_DIR}/bindings.cpp")
 
 set(scripts_hdr_parser "${CMAKE_CURRENT_SOURCE_DIR}/../python/src2/hdr_parser.py")
 
+set(JS_HELPER "${CMAKE_CURRENT_SOURCE_DIR}/src/helper.js")
```
Comment (huningxin) [inline modules/js/CMakeLists.txt]: `helpers.js`

**Review by huningxin — CHANGES_REQUESTED** [review]

**Comment by pancx** [comment]
@huningxin  A new commit for fixing the depends of the target doxygen.
 Add opencv.js to depends and delete the useless target of copy_js_assets.
Please review it.

File: modules/js/CMakeLists.txt (hunk 4 above)
Comment (huningxin) [reply modules/js/CMakeLists.txt]: @pancx , please change the file name from `helper.js` to `helpers.js`, as it contains several helper functions. Thanks!

**Comment by huningxin** [comment]
Looks good with one nit.

**Comment by pancx** [comment]
change filename helper.js to helpers.js in new commit

**Comment by huningxin** [comment]
looks good!

**Review by huningxin — APPROVED** [review]

**Pull request merged**